### ISSUE TITLE
New stake reward seems to be wrong

### ISSUE THREAD

**Issue by adam-635** [buildcore-io/core#1952]
@boldizsar-unchained - can you urgently look into this?

See this spreadsheet of current stakers: https://docs.google.com/spreadsheets/d/1qzStCW4s5nP9S2WgmCTZy7zrjddX0qzlvnaRvNOHiRU/edit#gid=713702813

```
There is definitely something wrong!

The following case is ONE user (0xbcde7a63a590fa0ef20097a2202f5e6134760d32) that staked for 1 year and got an okayish amount in the first week (266 SOON) but this week he only got 6 SOON ----> total rewards of 272 SOON and this wrong number matches the googlesheet:

![image.png](https://images.zenhubusercontent.com/54f851af4f7a08fbb7f242c5/9a42e5f8-80be-4def-8751-45254daa3835)

![image-1.png](https://images.zenhubusercontent.com/54f851af4f7a08fbb7f242c5/e4b4a0d5-ff85-4296-a994-5a8f69c0ffd0)

![image-2.png](https://images.zenhubusercontent.com/54f851af4f7a08fbb7f242c5/fc04e2ec-b240-4c98-aa30-16d94db81100)

Member id: 
https://soonaverse.com/member/0xacf8091920dbf56ba0bc8d5acc289cd0d85257bf/tokens 

This user was awarded ~23 tokens for their 11,620.24 Soon stake (1 yr) for example. My rewards for 20000 Soon at less than a year awarded ~257.0

In other words, mine went up and the user's went down, so it seems like there must be a problem, even if it matches the sheet 
```
